components/Header/index.tsx

```diff
@@ -1,13 +1,24 @@
 export default function Header({
   href,
   serviceName,
+  className = "yellow-border",
 }: {
   href?: string;
   serviceName?: string;
+  className?: string;
 }) {
+  const headerClass = "govuk-header--" + className;
+  const containerClass = "govuk-header__container--" + className;
+
```

Comment:
I think this approach is fine, but I wouldn't use a prop called `className` to achieve it - reason being, if I wanted to add an _additional_ class to the header for whatever reason, this means I have to do some classname injection to do it, e.g.

```jsx
<Header className="yellow-border ross-custom-class" />
```

Other thing you could so is enumerate the options in the type definition, e.g.

```jsx
borderType? : "yellow" | "blue" | "blue-alt"